### PR TITLE
integration test workflow

### PR DESCRIPTION
**Pull request by Floinko**

### CODE REVIEW

**Review by dominic-heil — COMMENTED** [review]
Copyright headers are missing and I think we should add a timeout to the workflow in case it gets stuck somewhere so that it does not run for forever. Once that is addressed you will get my approval.

The creation of the appwrite instance looks very good and will be very useful.

Regarding the integration test example, we might change that in a future sprint to cypress which is specialized for end-to-end-, unit- and integration-tests of websites. But that is just a comment. The example however is more then fine :D

File: .github/workflows/run_integration_tests.yml
Comment (dominic-heil) [inline .github/workflows/run_integration_tests.yml:33]: Is there a reason why this is commeted out?

File: backend/setup_appwrite/init-cli.sh
Comment (steinkohl) [inline backend/setup_appwrite/init-cli.sh:11]: Shouldn't be locale "de-DE" ? I think this also affects timestamps

**Review by steinkohl — APPROVED** [review]
Nice work in general.
But I would prefer passwords to be stored in githubs env varaibles, even if they are only intended for testing. If somebody misuses this script to deploy their production server, they will have a very fun time.

**Review by dominic-heil — APPROVED** [review]
LGTM

**Comment by tuilagio** [comment]
can this PR be merged? I need some of these changes for my issue ...

**Pull request merged**
